### ISSUE TITLE
Focused item is getting faded in Tizen & WebOS

### ISSUE THREAD

**Issue by SandeepKharbanda** [pavjacko/renative#506]
Whenever I move my focus on to any `TouchableOpacity` button then the button is getting faded. I have already set its activityOpacity prop to 1 i.e`activityOpacity={1}` but still, It's getting faded.

**Comment by SandeepKharbanda** [comment]
Any update on this as I need to submit the app to the respective store?

**Comment by pavjacko** [comment]
@SandeepKharbanda fix for this is coming in `0.30.0` (upcoming release)
you can test latest alpha preview `0.30.0-rc2`

you override default value for Button with `activeOpacity` as per example excerpt from hello-world below:

```
<Button
    iconFont="fontAwesome"
    focusKey="github"
    iconName="github"
    activeOpacity={1}
/>
```
